lib/core/Manager.js

```diff
@@ -112,7 +114,12 @@ Manager.prototype.resolve = function () {
     // Otherwise, fetch each target from the repository
     // and let the process roll out
     } else {
-        this._targets.forEach(this._fetch.bind(this));
+        this._targets.forEach(function (target) {
+            var name = target.name || target.source;
+            if (!this._requested.length || this._requested.indexOf(name) > -1) {
```

Comment:
Just to try to be coherent with the rest, it's better to use [mout.array.contains](http://moutjs.com/docs/v0.1.0/array.html#contains).